packages/discovery-provider/ddl/migrations/0120_delete_listen_streak_endless_challenges.sql

```diff
@@ -0,0 +1,3 @@
+BEGIN;
```

Comment:
Did you pick up a reverted PR? There's a bunch of stuff in here that seems unrelated to your changes.

Reply:
yaaa mb. Fixed now.